types/leaflet/index.d.ts

```diff
@@ -667,7 +667,9 @@ export function canvas(options?: RendererOptions): Canvas;
  * added/removed on the map as well. Extends Layer.
  */
 export class LayerGroup<P = any> extends Layer {
-    constructor(layers?: Layer[]);
+    constructor(layers?: Layer[], options?: LayerOptions);
+    initialize(layers?: Layer[], options?: LayerOptions): this;
```

Comment:
@atd-schubert what do you think about adding `initialize(....args: Array<any>): this` to `Class` to support the general case and leaving this overload in place for the specific case. I haven't tested if typescript will be happy with it but if it does, it will provide reasonable balance between type safety and use-at-your-own-risk.